### PR TITLE
Use source branch for ci

### PR DESCRIPTION
**Pull request by Roming22**
Add an option to dev_setup.sh to deploy the current branch.
This option should make it easy to test the source branch in PRs.

Signed-off-by: Romain Arnaud <rarnaud@redhat.com>

Validation of the PR needs to happen in throw away PR #471.

### CODE REVIEW

**Review by bnallapeta — APPROVED** [review]

File: developer/openshift/dev_setup.sh [hunk 1]
```diff
@@ -238,9 +271,20 @@ install_pipeline_service() {
   "$PROJECT_DIR/operator/images/access-setup/content/bin/setup_work_dir.sh" \
     ${DEBUG:+"$DEBUG"} \
     --work-dir "$WORK_DIR" \
-    --kustomization "$GIT_URL/developer/openshift/gitops/argocd?ref=$GIT_REF" |
+    --kustomization "git::$GIT_URL/developer/openshift/gitops/argocd?ref=$GIT_REF" |
```
Comment (Mo3m3n) [inline developer/openshift/dev_setup.sh:274]: shouldn't we also use PROJECT_DIR here too when `USE_CURRENT_BRANCH` is set ?

**Review by Mo3m3n — CHANGES_REQUESTED** [review]

File: developer/openshift/dev_setup.sh (hunk 1 above)
Comment (Roming22) [reply developer/openshift/dev_setup.sh:274]: No. What's happening is that we set everything as usual, but before running the install, we go in and modify the content, which is what's happening line 277.

**Pull request merged**